### PR TITLE
Update for Icons 7.1.0

### PR DESCRIPTION
**Pull request by danirabbit**

### CODE REVIEW

**Review by ryonakano — COMMENTED** [review]
Looks good and it worked, but don't we also add "computer-fail", which is also listed as a new icon in the icon's release note?

**Comment by danirabbit** [comment]
@ryonakano I excluded "computer-fail" since it's meant to be used by the shell to indicate a crash or failure of the system. I don't think it makes sense for this icon to appear in an application, but I don't super mind adding it if there is demand for something like this

**Comment by ryonakano** [comment]
Ah I see, if it's meant to be used in the system and not intended to be used in the apps by third-parties, then we should exclude that icon from this app. :+1:

**Review by ryonakano — APPROVED** [review]

File: data/iconbrowser.appdata.xml.in [hunk 1]
```diff
@@ -40,6 +40,18 @@
   <update_contact>contact_at_elementary.io</update_contact>
 
   <releases>
+     <release version="2.1.0" date="2022-10-06">
```
Comment (ryonakano) [inline data/iconbrowser.appdata.xml.in:43]: Ah, don't forgot to update the date if you would before merging :)

**Pull request merged**